engine/moduleManager/modules/input/KeyEvent.hpp

```diff
@@ -0,0 +1,24 @@
+//
+// Created by 11896 on 26/11/2024.
+//
+
+#ifndef BLOCKYENGINE_ENGINE_MODULEMANAGER_MODULES_INPUT_KEYEVENT_HPP_
+#define BLOCKYENGINE_ENGINE_MODULEMANAGER_MODULES_INPUT_KEYEVENT_HPP_
+
+#include "SDL_keycode.h"
+#include "KeyInput.hpp"
+
+enum class KeyState {
```

Comment:
"class" kan weg. zolang je geen van plan bent om in je enum enge methodes te gebruiken is die overhead overbodig